package.nls.de.json

```diff
@@ -26,9 +26,11 @@
   "cmd.updateSettings.error.readGistFail": "Sync :  Gist-ID: {0} UNLESBAR.",
   "cmd.updateSettings.error.gistNotSave": "Sync : GIST NICHT GESICHERT",
   "cmd.downloadSettings.title": "Sync : Einstellungen herunterladen",
+  "cmd.downloadSettings.info.downloaded": "Sync : Download Complete.",
   "cmd.downloadSettings.info.readdingOnline": "Sync : Lese Online-Einstellungen.",
   "cmd.downloadSettings.info.gotLatestVersion": "Sync : Du hast bereits die aktuellesten Einstellungen.",
   "cmd.downloadSettings.error.removeExtFail": "Sync : Kann manche Erweiterungen nicht entfernen.",
+  "cmd.downloadSettings.error.unableSave": "Sync : Unable to save extension settings file.",
```

Comment:
TODO: Waiting to be translated

Reply:
Die Erweiterungseinstellungen konnten nicht gespeichert werden

Reply:
@Karlito88 create PR for this :) if some keys are missing